LEAF_Nexus/api/controllers/TagController.php

```diff
@@ -26,7 +26,7 @@ public function get($act)
         });
 
         $this->index['GET']->register('tag/[text]/parent', function($args) use ($tag) {
-            return $tag->getParent($args[0]);
+            return $tag->getParent($tag->sanitizeInput($args[0]));
```

Comment:
This should use the sanitize functions in `XSSHelpers.php`. `Tag.php` uses the sanitize function in `Data.php`, which is an old version of the sanitation functions.